concourse/pipelines/autodelete-cloudfoundry.yml

```diff
@@ -6,16 +6,25 @@ resources:
       stop: 6:00 -0000
       interval: 2h
 
+  - name: bosh-secrets
```

Comment:
And now, I will be evil and ask you to raise a different PR for this last commit (942f5e0f129ae21378db73c63fc4d5ad6dacaead), referring to the story [#112670769](https://www.pivotaltracker.com/story/show/112670769)  :laughing: 

It is easy, only do:
```
git checkout 112581575_skip_release_download
git checkout -b 112670769_get_bosh_password_in_pipeline # create a new branch
git push

git checkout 112581575_skip_release_download
git reset HEAD^1 --hard # drop last commit
git push -f origin 112581575_skip_release_download

```

And do the PR for 112670769_get_bosh_password_in_pipeline